### PR TITLE
bug: Fixes writing binary data to influxdb...

### PR DESCRIPTION
**Pull request by mangodan2003**
... as it was it was writing the error message produced trying to parse binary as utf-8

bug

### CODE REVIEW

File: v1/src/lib.rs [hunk 1]
```diff
@@ -443,7 +443,7 @@ impl Storage for InfluxDbStorage {
         // encode the value as a string to be stored in InfluxDB, converting to base64 if the buffer is not a UTF-8 string
         let (base64, strvalue) = match payload.try_to_string() {
             Ok(s) => (false, s),
-            Err(err) => (true, b64_std_engine.encode(err.to_string()).into()),
+            Err(err) => (true, b64_std_engine.encode(payload.to_bytes()).into()),
```
Comment (JEnoch) [inline v1/src/lib.rs:446]: Shall be `Err(_)` instead of `Err(err)` , otherwise rustc complains that `err` id not used.

**Review by JEnoch — CHANGES_REQUESTED** [review]

**Comment by JEnoch** [comment]
Well spotted! Thanks for fixing it!
I'll merge as soon as my comment is addressed.

File: v1/src/lib.rs (hunk 1 above)
Comment (mangodan2003) [reply v1/src/lib.rs:446]: doh! indeed, ill get that sorted, thanks

**Comment by gabrik** [comment]
Thanks @mangodan2003 for your contribution, however in order to be able to merge this you need to sign the Eclipse Contribution Agreement: https://www.eclipse.org/legal/eca/

**Comment by mangodan2003** [comment]
RE the ECA, I have signed this previously and having checked it says it is still valid, but I have re-submitted it anyway having updated the info slightly.

**Comment by gabrik** [comment]
> RE the ECA, I have signed this previously and having checked it says it is still valid, but I have re-submitted it anyway having updated the info slightly.

Seems that the email used to sign the commits does not match the one used to sing the ECA.

**Comment by mangodan2003** [comment]
I had gotten the impression it went by the github username, I've updated my email address on the ECA site. hopefully that resolves it.

**Comment by gabrik** [comment]
I re-triggered the validation, seems ok now

**Review by JEnoch — APPROVED** [review]
Thanks for the update, LGTM now !

**Pull request merged**